### PR TITLE
Fix ADEV

### PR DESCRIPTION
**Pull request by alan-agius4**

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit b80b46215c805b7456b2e3013cbe0c1049df2699.

File: adev/tools/local_deps/filter_external_npm_deps.bzl [hunk 1]
```diff
@@ -6,7 +6,7 @@ load("@build_bazel_rules_nodejs//:providers.bzl", "ExternalNpmPackageInfo", "Lin
 def _filter_external_npm_deps_impl(ctx):
     problematic_paths = ["external/npm/node_modules/%s" % pkg for pkg in ctx.attr.angular_packages]
     package_name = ctx.attr.target.label.package
-    has_problematic_transitive_dep = package_name.startswith("@angular-devkit/")
+    has_problematic_transitive_dep = package_name.startswith("@angular-devkit/") or package_name.startswith("@angular/build")
```
[the remark this answers is not in the document]
Comment (alan-agius4) [reply adev/tools/local_deps/filter_external_npm_deps.bzl:9]: None of the tooling packages have a dependency on the FW packages, only peer deps.

[the remark this answers is not in the document]
Comment (devversion) [reply adev/tools/local_deps/filter_external_npm_deps.bzl:9]: Peer deps matter as well. Bazel doesn't distinguish between peer deps or not, from a build graph perspective

**Pull request closed without merging**